modules/core/shared/src/main/scala/coursier/core/Resolution.scala

```diff
@@ -1193,10 +1193,10 @@ object Resolution {
           .flatMap(projectCache(_)._2.dependencies)
       )
       .withDependencyManagement(
-        depMgmt.values.toSeq
```

Comment:
btw this was to remove a `Stream` as explained in 1cb8f3d15358fcb856f86e28261a0c63fec0c6c9

Reply:
any reason you reverted this? Can I include it in another PR?

Reply:
> Can I include it in another PR?

Sure, go ahead! It was somewhat unrelated to the other changes of this PR (memo-izing some data class instances), I preferred not to include it here.